package.json

```diff
@@ -11,13 +11,14 @@
         "check-types": "tsc --noemit",
         "eslint": "eslint src",
         "lint": "pnpm run eslint && pnpm run check-types",
-        "format": "npx prettier --write \"src/**/*.{ts,tsx,js,jsx,json,css,scss,md}\"",
-        "check-format": "npx prettier --check \"src/**/*.{ts,tsx,js,jsx,json,css,scss,md}\""
+        "format": "prettier --write \"src/**/*.{ts,tsx,js,jsx,json,css,scss,md}\"",
+        "check-format": "prettier --check \"src/**/*.{ts,tsx,js,jsx,json,css,scss,md}\""
     },
     "type": "module",
     "devDependencies": {
         "@capacitor/assets": "^2.0.4",
         "@capacitor/cli": "^5.2.2",
+        "@ianvs/prettier-plugin-sort-imports": "^4.1.0",
```

Comment:
this seems great but as a security concern I try to pretty conservative with adding dependencies, same with prettier-plugin-tailwind

would much prefer if these could just be binary tools instead of dependencies!

Reply:
I understand (but `prettier-plugin-tailwind` is made by tailwind labs, if this dependency can be compromised `tailwindcss` can be too)

Am I right to assume that it is to protect your working environment (and the team's) and not mine for example, or the cloud flare page build step, etc ?

If so I might have a solution with optional (well peer) deps, checks if they're installed in prettier (or create a wrapper config if too hard) and maybe to get fancy a GitHub action with weekly PRs so that you don't even have to think about it.

Reply:
No matter what I do (optional, peer, optional peer) `pnpm` installs all reps by default.. I can still do the action though